.github/workflows/openapi.yml

```
@@ -0,0 +1,37 @@
+on:
+  push:
+    branches: [ main, feature/openapi-spec-issue-50 ]
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  generate:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v4
+    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)